src/test/java/com/google/api/codegen/testdata/java_sample_library.baseline

```diff
@@ -0,0 +1,51 @@
+============== file: build.gradle ==============
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+}
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+description = 'Sample client library for google-cloud-library-v1'
+group = "com.google.api"
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+repositories {
+  mavenCentral()
+  mavenLocal()
+}
+
+dependencies {
+  compile project(":grpc-google-cloud-library-v1"),
+    'com.google.api:gax:1.0.0',
+    'com.google.api:gax-grpc:0.18.0',
+    'commons-cli:commons-cli:1.4',
+    'commons-lang:commons-lang:2.6',
+    'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork26'
```

Comment:
I feel like this needs a comment telling the user they have to keep it in sync with the version that grpc needs.